### PR TITLE
Further processing for certain strings

### PR DESCRIPTION
**Pull request by JoshSinghONS**
What
Added further processing of string if over a certain length i.e. if the string containing the date range holds two months and a year.

Fix needed because original code would move year provided by the start month and put it after the end month resulting in 2020 DEC - FEB becoming DEC - FEB 2020 when it should be DEC - FEB 2021.

Further discussions has resulted in the desired output being DEC 2020 - FEB 2021 and the code added reflects that desire.

How to review
LGTM / Feel free to pull and check locally.

Who can review
Someone familiar with GO

### CODE REVIEW

File: helper/date_period_format_test.go [hunk 1]
```diff
@@ -14,6 +14,11 @@ func TestDatePeriodFormat(t *testing.T) {
 		got := helper.DatePeriodFormat("2010 JUN-SEP")
 		So(got, ShouldEqual, want)
 	})
+	Convey("Given a time-series monthly string spanning two different years", t, func() {
+		want := "Dec 2010 - Jan 2011"
+		got := helper.DatePeriodFormat("2010 DEC-JAN")
```
Comment (goofballLogic) [inline helper/date_period_format_test.go:19]: this format isn't want you indicated in the comments as desired:

> Further discussions has resulted in the desired output being DEC 2020 - FEB 2021 and the code added reflects that desire.

**Review by goofballLogic — COMMENTED** [review]
comments / code do not appear to align

File: helper/date_period_format_test.go (hunk 1 above)
Comment (JoshSinghONS) [reply helper/date_period_format_test.go:19]: Whatcha talkin bout willis?

Given 2010 DEC-JAN we want Dec 2010 - Jan 2011 or are you getting hung up on the capitals?

Comment (goofballLogic) [reply helper/date_period_format_test.go:19]: oooooo stupid me misread. got/want (I'm used to actual/expected)

**Review by goofballLogic — APPROVED** [review]

**Review by rafahop — APPROVED** [review]

**Pull request merged**